tests/foreman/cli/test_repositories.py

```diff
@@ -0,0 +1,46 @@
+"""Test module for Repositories CLI.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:team: Phoenix-content
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9])
```

Comment:
```suggestion
@pytest.mark.rhel_ver_match('[^6]')
```

Reply:
This would also parametrize the test to use `_fips` clients. Is it desired?

Reply:
not necessary here